src/main/java/org/opensearch/security/dlic/rest/api/SecuritySSLCertsAction.java

```diff
@@ -91,8 +89,6 @@ public SecuritySSLCertsAction(final Settings settings,
                                   final boolean certificatesReloadEnabled) {
         super(settings, configPath, controller, client, adminDNs, cl, cs, principalExtractor, privilegesEvaluator, threadPool, auditLog);
         this.securityKeyStore = securityKeyStore;
-        this.restApiAdminPrivilegesEvaluator =
```

Comment:
Oh! That was silly, thanks for fixing

Reply:
My bad. Did not notice it in my initial PR